### PR TITLE
Repo Overhaul

### PR DESCRIPTION
**Pull request by SoftCreatR**
- Move participants into separate files and folders
- Remove script validate-alphabetical.py
- Add script update_readme.py
- Add workflow
- Add JSON Schema for participant file validation
- Update existing workflow
- Update README
- Add generic Hacktoberfest logo
- Rename tshirt to shirt
- Add CODE_OF_CONDUCT.md
- Rename contributing.md to CONTRIBUTING.md

### CODE REVIEW

**Comment by SoftCreatR** [comment]
Btw: After merging, the CONTRIBUTING.md needs your attention @benbarth

File: .github/workflows/UpdateReadme.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+name: Update README
+
+on:
+  push:
+    paths:
+      - 'participants/**/*.yml'
+  schedule:
+    - cron: '0 0 1 1 *'
```
Comment (benbarth) [inline .github/workflows/UpdateReadme.yml:9]: This runs once every year. Is that intentional?

File: .github/workflows/UpdateReadme.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  workflow_dispatch:
```
Comment (benbarth) [inline .github/workflows/UpdateReadme.yml:10]: Is this okay to be empty/blank?

File: .github/workflows/UpdateReadme.yml (hunk 1 above)
Comment (SoftCreatR) [reply .github/workflows/UpdateReadme.yml:9]: Yes, it is. See https://github.com/benbarth/hacktoberfest-swag/pull/118#issuecomment-701383741

File: .github/workflows/UpdateReadme.yml (hunk 2 above)
Comment (SoftCreatR) [reply .github/workflows/UpdateReadme.yml:10]: Yes, it is.

**Comment by benbarth** [comment]
🏆  @SoftCreatR wins "Best PR."

**Comment by SoftCreatR** [comment]
@benbarth You may try out the new workflow by adding a participant yml file ;) Only if that works on the 1st try, i made a good job :octocat:

**Comment by benbarth** [comment]
I received an email:

> The page build failed for the `master` branch with the following error:
> 
> Page build failed. For more information, see https://docs.github.com/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#troubleshooting-build-errors.
> 
> For information on troubleshooting Jekyll see:
> 
>   https://docs.github.com/articles/troubleshooting-jekyll-builds

**Comment by SoftCreatR** [comment]
I've seen this already. Unfortunately, there's no reason given for the failure.

You may try triggering the build again.

**Comment by benbarth** [comment]
I ran the workflow again and everything succeeded... unfortunately the Jekyll [website](https://benbarth.github.io/hacktoberfest-swag/) isn't updating.

**Comment by SoftCreatR** [comment]
I found a hand full of issues within the README file. I'll try to fix them. Maybe, that does the trick.

**Comment by benbarth** [comment]
🤞

**Comment by SoftCreatR** [comment]
> I ran the workflow again and everything succeeded... unfortunately the Jekyll [website](https://benbarth.github.io/hacktoberfest-swag/) isn't updating.

Hitting the "Re-run jobs" button isn't the correct solution ;) This doesn't trigger the gh-pages workflow. However, i've created another PR with some modifications to the README file. Let's see, if that works.

**Comment by SoftCreatR** [comment]
Yay!

**Pull request merged**